### DISCUSSION TITLE
Expo Router v2 + Nativewind v2 🚀

### DISCUSSION THREAD

**Discussion by redkean** [nativewind/nativewind#545] [General]
#### With the recent beta support for [metro css bundling](https://docs.expo.dev/versions/latest/config/metro/#css), this library works on native and web platform without the need to do the fix on https://github.com/marklawlor/nativewind/issues/470.
---

Following Expo's configuration of tailwind for metro [here](https://docs.expo.dev/versions/latest/config/metro/#tailwind) you should be good to go. 

_Metro CSS only works on web though, but paired with this library, **this can be powerful!**_

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
